packages/carousel/src/styledComponents/Carousel.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+const scale = 1.219;
+
+const Container = styled.div`
+  position: relative;
+  ul {
+    position: relative;
+    left: 50%;
+    margin-left: ${props => -props.itemConfig.width * scale / 2}px;
+    height: ${props =>
+      props.itemConfig.height * scale + props.itemConfig.margin * 2}px;
+    list-style: none;
+    padding: 0;
+    display: inline-flex;
+    align-items: center;
+
+    li {
+      margin: ${props => props.itemConfig.margin}px;
+      transition: all 2s;
```

Comment:
can we receive `2s` from configs?

Imagine that `config.displayTime` is `1000ms`. We should be able to configure transition to use `0.5s` for instance

Reply:
https://cravefoodservices.slack.com/files/U4KA0RLBD/F9VLURPCP/carousel.mov